backend/src/main/java/com/redhat/cloud/notifications/recipients/itservice/pojo/request/ITUserRequest.java

```diff
@@ -0,0 +1,42 @@
+package com.redhat.cloud.notifications.recipients.itservice.pojo.request;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class ITUserRequest {
+
+    public By by;
+    public Include include;
+
+    public ITUserRequest(String accountId, boolean adminsOnly) {
+        final By by = new By();
+        AllOf allOf = new AllOf();
+        allOf.status = "enabled";
+        allOf.ebsAccountNumber = accountId;
+
+        if (adminsOnly) {
+            PermissionCode permissionCode = new PermissionCode();
+            permissionCode.value = "admin:org:all";
+            permissionCode.operand = "eq";
+            allOf.permissionCode = permissionCode;
+        }
+        by.allOf = allOf;
+
+        this.by = by;
+
+        Include include = new Include();
+
+        include.allOf = List.of("authentications", "personal_information");
+        List<AccountRelationship> accountRelationships = new LinkedList<>();
+        AccountRelationship accountRelationship1 = new AccountRelationship();
+        accountRelationship1.allOf = List.of("primary_email");
+
+        final By__1 by1 = new By__1();
+        by1.active = true;
+        accountRelationship1.by = by1;
+        accountRelationships.add(accountRelationship1);
+        include.accountRelationships = accountRelationships;
```

Comment:
`include` seems constant. Can we avoid rebuilding it each time we need to send a request?